README.md

```diff
@@ -12,17 +13,23 @@
 - Run "./npm i @angular/cli"
 - Run "./npm i"
 - Run "./mvnw generate-resources"
+
+# IntelliJ
 - Open the project in intellij
-- Build src/main/java/AuthenticationProxyApplication.java
-- Run src/main/java/AuthenticationProxyApplication.java
+- Change the user to postgres and password to the password for that superuser in application.properties
+- Build and run src/main/java/AuthenticationProxyApplication.java
 
-- ! For the application to run the capstone_user must have permission to SELECT from both tables !
+# pgAdmin
+- Go into pgAdmin, navigate to the capstone_user user under Login/Group Roles under the PostgreSQL 15 server. 
+- Right-click and select Properties, go to the Privileges tab, and enable the 'Can initiate streaming replication and backups?' privilege, then click Save
+- Open up the Capstone database under PostgreSQL 15, then navigate to the 2 tables that were created under the Public schema
+- If the tables aren't there, click 'Refresh' by right-clicking anywhere
+- On both the users and user_roles tables, right-click and select Properties
+- Within the Security tab, click the add (+) button in the top-right, then select the Grantee as capstone_user and the Privileges as Select (r) in the dropdowns
+- Click Save
 
-## Angular
-- Every time an update is made to the Angular side of the project, you must rebuild the project with "./mvnw generate-resources" and re-run it
+# IntelliJ pt. 2
+- Head back into the application.properties file and change the username and password back to capstone_user and ${ENV_NAME} respectively
```

Comment:
its fine to say ${capstone_user}, not a security issue and only our team needs this. Just saying this so there is no confusion when team members go through this README